contributing/code/pull_requests.rst

```diff
@@ -429,6 +429,68 @@ After the `Psalm phar is installed`_, the analysis can be run locally with:
 
     $ psalm.phar src/Symfony/Component/Workflow
 
+Automated Tests
+~~~~~~~~~~~~~~~
+
+A series of automated tests will run when submit the pull request. These
+test the code under different conditions, to be sure nothing important is
+broken. Test failures can be unrelated to your changes. If you think this
+is the case, you can check if the target branch has the same errors and
+leave a comment on your PR.
+
+Otherwise, the test failure might be caused by your changes. The following
+test scenarios run on each change:
+
+``PHPUnit / Tests``
+    This job runs on Ubuntu using multiple PHP versions (each in their
+    own job). These jobs run the testsuite just like you would do locally.
+
+    A failure in these jobs often indicate a bug in the code.
```

Comment:
indicates

Reply:
"these jobs" is plural, so I think "indicate" is correct.

I did a bit of a plot twist in this description, changing from singular to plural after mentioning "multiple PHP versions".

Reply:
the subject is "A failure", which is singular, isn't it?